wls-vorfaelleundvorkommnisse-service/src/main/resources/application-local.yml

```diff
@@ -0,0 +1,2 @@
+server:
+  port: 39152
```

Comment:
bitte nächsten freien Port verwenden. `39152` soll durch den Auth-Service verwendet werden.  Hintergrund dazu ist [dieser ADR](https://it-at-m.github.io/Wahllokalsystem/technik/adr/adr-improve-getting-started-of-services.html#entscheidung)